crates/sui-indexer-alt-jsonrpc/src/metrics/middleware.rs

```diff
@@ -102,6 +104,7 @@ where
                 failed: self.layer.metrics.requests_failed.clone(),
             }),
             method,
+            params: format!("{:?}", request.params()),
```

Comment:
```suggestion
            params: request.params().as_str(),
```

Reply:
this fix per se would not work, one reason is that this would return type of Option<&str>, a deeper err is that `params()` create a new temp value and cannot be returned 
```
	pub fn params(&self) -> Params {
		Params::new(self.params.as_ref().map(|p| RawValue::get(p)))
	}
```

I changed the type to `Option<Cow<'a, RawValue>>` and pass along `params` field instead of the method to minimize clone instead.